app/utils/validators/forms/FieldValidator.scala

```diff
@@ -136,4 +136,9 @@ object FieldValidator {
 
   private val emailPattern = Pattern.compile("""^\S+@\S+$""")
   val isValidEmail: String => Boolean = (name: String) => emailPattern.matcher(name).matches()
+
+  val REGEX = "^(GB|gb)[\\d]+$"
```

Comment:
Please use name `eoriRegex`, because regex is too wide name for something in `FieldValidator` which should support a lot of different validation.

Additionally in Scala vals are declared using lower case letters. To show that this is constant value it shuld have just first character as capital letter.

Reply:
Fixed